public/css/showoff.css

```diff
@@ -96,14 +105,6 @@
     margin-right:auto;
 }
 
-.slide .center {
```

Comment:
![screen shot 2015-06-08 at 4 06 22 pm](https://cloud.githubusercontent.com/assets/3277190/8047303/5b06c2d0-0df8-11e5-833d-3a4110741659.png)

As far as I can tell, using the showoff example slides, it looks exactly the same.  The transform is doing the same thing as this was, so this became redundant